.github/workflows/lint.yml

```diff
@@ -17,9 +17,10 @@ jobs:
         with:
           go-version-file: go.mod
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v7
+        uses: golangci/golangci-lint-action@v8
         with:
-          version: v2.0.2
+          version: v2.1.6
+          args: '--timeout 10m'
```

Comment:
Why?

Reply:
the deadline was part of the config, since the new golang-lint action, we need the timeout to be in args